README.md

```diff
@@ -67,12 +91,12 @@ webpack's equivalent of browserify transforms and RequireJS plugins is a **loade
 module.exports = {
   entry: './main.js',
   output: {
-    filename: 'bundle.js'       
+    filename: 'bundle.js'
   },
   module: {
     loaders: [
       { test: /\.coffee$/, loader: 'coffee-loader' },
-      { test: /\.js$/, loader: 'babel-loader' }
+      { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader'}
```

Comment:
I would highly recommend to go with white-listing and use `include` instead of `exclude`, so babel will transpile only things we intended to transpile.